packages/flutter/lib/src/material/about.dart

```diff
@@ -625,3 +1009,658 @@ Widget _defaultApplicationIcon(BuildContext context) {
+          ),
+          body: _masterPanel(context),
+          floatingActionButton: widget.floatingActionButton,
+        ),
+        // Detail view stacked above main scaffold and master view.
+        SafeArea(
+          child: Padding(
+            padding: EdgeInsetsDirectional.only(
+              start: masterViewWidth - _kCardElevation,
+              end: widget.floatingActionButton == null
+                  ? detailPageFABlessGutterWidth
+                  : detailPageFABGutterWidth,
+            ),
+            child: ValueListenableBuilder<Object>(
+              valueListenable: _detailArguments,
+              builder: (BuildContext context, Object value, Widget child) {
+                return AnimatedSwitcher(
+                  transitionBuilder:
+                      (Widget child, Animation<double> animation) =>
+                      const FadeUpwardsPageTransitionsBuilder()
+                          .buildTransitions<void>(
+                          null, null, animation, null, child),
+                  duration: const Duration(milliseconds: 500),
+                  child: Container(
+                    key: ValueKey<Object>(value ?? widget.initialArguments),
+                    constraints: const BoxConstraints.expand(),
+                    child: _DetailView(
+                      builder: widget.detailPageBuilder,
+                      arguments: value ?? widget.initialArguments,
+                    ),
+                  ),
+                );
+              },
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  ConstrainedBox _masterPanel(BuildContext context, {bool needsScaffold = false}) {
+    return ConstrainedBox(
+      constraints: BoxConstraints(maxWidth: masterViewWidth),
+      child: needsScaffold
+          ? Scaffold(
+              appBar: AppBar(
+                title: widget.title,
+                actions: widget.actionBuilder(context, _ActionLevel.top),
+                leading: widget.leading,
+                automaticallyImplyLeading: widget.automaticallyImplyLeading,
+                centerTitle: widget.centerTitle,
+              ),
+              body: widget.masterViewBuilder(context, true),
+            )
+          : widget.masterViewBuilder(context, true),
+    );
+  }
+}
+
+class _DetailView extends StatelessWidget {
+  const _DetailView({
+    Key key,
+    @required _DetailPageBuilder builder,
+    Object arguments,
+  })  : assert(builder != null),
+        _builder = builder,
+        _arguments = arguments,
+        super(key: key);
+
+  final _DetailPageBuilder _builder;
+  final Object _arguments;
+
+  @override
+  Widget build(BuildContext context) {
+    if (_arguments == null) {
+      return Container();
+    }
+    final double screenHeight = MediaQuery.of(context).size.height;
+    final double minHeight = (screenHeight - kToolbarHeight) / screenHeight;
+
+    return GestureDetector(
+      onTap: () {
+        print('draggable');
+      },
```

Comment:
This print (and probably the entire gesture detector?) should probably be removed?

Reply:
Ooh, yes, good point.  @TonicArtos can you take care of that?

Reply:
Yes. It is late here. I'll get caught up with up and get the fix sorted tomorrow.

Reply:
OK, no huge rush: most of us will be on holiday until Monday (it's Independence Day in the U.S.).

Reply:
I went through my journal. It turns out the GestureDetector is left from when I was looking at #59741. I forgot to remove it once I figured what was happening.

Reply:
PR #60730 Remove superfluous GestureDetector.